rest-api/src/main/resources/paths/studies/v5_studies_studyId_participants_userId_adherence_instanceGuid_eventTimestamp_postprocessing.yml

```diff
@@ -0,0 +1,30 @@
+post:
+    operationId: updateAdherencePostProcessingAttributes
+    summary: Update just the specific post-processing attributes on the adherence record.
+    tags:
+        - Study Adherence
+        - _For Researchers
+        - _For Study Coordinators
```

Comment:
Can add tags for Developers and Study Designers since they can also access this.